velox/dwio/common/QplJobPool.cpp

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
I'm also wondering is this class belongs under dwio. For example, we would probably want to use this for shuffles as well.

Reply:
@yaqi-zhao Any possible to add this logic to QPL library? So in Velox we can have a clean ZLIB_IAA codec interface?
If not can we move this logic to IAA compression codec? 

@pedroerp The logic is private to IAA codec only. If shuffle or spill use IAA, it should use the codec interface instead of the interfaces in the file.